test/warehouse.test.js

```diff
@@ -309,6 +323,7 @@ function itemToObject(item) {
             ? getISODateNoMs(new Date(item.no_update_after))
             : undefined,
         item_id: item.item_id.toNumber(),
+        name: item.name.toString(),
```

Comment:
is the .toString() here required? `.toNumber()` is necessary since item_id and quantity are `BigNumber` objects (https://github.com/MikeMcl/bignumber.js/) but strings should just be JS strings already. Can you confirm?